install/fo-apache.conf

```diff
@@ -5,8 +5,14 @@ Alias /repo /usr/share/fossology/www/ui
 <Directory "/usr/share/fossology/www/ui">
 	AllowOverride None
 	Options FollowSymLinks MultiViews
-	Order allow,deny
-	Allow from all
+	
+	# Order and Allow are for Apache httpd 2.2
+	# Order allow,deny
+	# Allow from all
```

Comment:
One should prefer the approach working on both version. This is described in the [comment above](https://github.com/fossology/fossology/pull/734#issuecomment-273523560).